### PR TITLE
refactor(backend): inject prisma via contextValue

### PR DESCRIPTION
**Pull request by roschaefer**
Motivation
----------
We already have `prisma` as `dataSource` in our `ContextValue`. This PR will import them in our resolvers.

We're not fully done yet, there are imports of `prisma` elsewhere. However, it's good for now. The code of the resolvers get copy+pasted, so it's important to refactor this part already.

How to test
-----------
1. `npm run test:unit`
2. All tests pass

### CODE REVIEW

File: backend/src/prisma.ts [hunk 1]
```diff
@@ -75,4 +75,12 @@ const userWithProfile = Prisma.validator<Prisma.UserDefaultArgs>()({
 
 type UserWithProfile = Prisma.UserGetPayload<typeof userWithProfile>
 
-export { prisma, UsersWithMeetings, UserWithMeeting, UserWithProfile }
+type DeamMallPrismaClient = typeof prisma
+
+export {
+  prisma,
+  DeamMallPrismaClient as PrismaClient,
```
Comment (Bettelstab) [inline backend/src/prisma.ts:82]: Seems to be unused?

**Review by Bettelstab — APPROVED** [review]
I think the overall motivation is better testability because it's easier to mock prisma, correct?
Behaviour should be as before, so all good.
Have a look at my comment, to see if that change is needed.

**Review by ulfgebhardt — COMMENTED** [review]
We can merge this - but again completely offtopic

File: backend/src/prisma.ts (hunk 1 above)
Comment (roschaefer) [reply backend/src/prisma.ts:82]: There is another import of `PrismaClient` in this file, that's why it's renamed here.

`PrismaClient` is imported in several locations in this PR.

**Pull request merged**